src/settings/components/general-settings/module-version-settings.tsx

```diff
@@ -37,12 +37,17 @@ const ModuleVersionSettings: FC = () => {
 		() => apps.filter((app) => app.name !== SEARCH_APP_ID && app.name !== SETTINGS_APP_ID),
 		[apps]
 	);
+	const sectionTitle = useMemo(
+		() => t(settingsSubSections[5].label, settingsSubSections[5].fallback),
+		[t]
+	);
 	return (
 		<>
 			<FormSubSection
-				label={t('module.app.version', 'Application versions')}
+				label={sectionTitle}
 				minWidth="calc(min(100%, 512px))"
 				width="50%"
+				id={sectionTitle.replace(/\s/g, '')}
```

Comment:
instead of adding multiple memorised value you can simply add another key with name of id in the settings subsection and use the memo in the following way to find

`export const settingsSubSections = [
	{
		label: 'settings.general.theme_options',
		fallback: 'Theme Options', 
                 key:'theme_option',
                id:theme_option (or any relevant id)
	}]
const sectionTitle= useMemo(
		() => find(settingsSubSection,{key:"SectionName"}),[dependencies]);`
		
and use it as label: sectionTitle.label ,id:sectionTitle.id something like this